br/pkg/lightning/checkpoints/checkpoints.go

```diff
@@ -51,6 +51,8 @@ const (
 	CheckpointStatusMaxInvalid      CheckpointStatus = 25
 	CheckpointStatusLoaded          CheckpointStatus = 30
 	CheckpointStatusAllWritten      CheckpointStatus = 60
+	CheckpointStatusDupDetected     CheckpointStatus = 70
```

Comment:
not found the saveStatusCheckpoint of this status, is this expected?

Reply:
Fixed in https://github.com/pingcap/tidb/pull/42647/commits/9d92a2bd2e1fcccf26d1fa5625cb9acea98409e0. The test will be added later.

Reply:
Test is added https://github.com/pingcap/tidb/pull/42647/commits/3543a67f67a7fd73d33acf211e581b73f383dad8.